pkg/imageserver/webdav.go

```diff
@@ -13,31 +13,30 @@ import (
 )
 
 const (
-	// CHROOT_SERVE_PATH is the path to server if we are performing a chroot
+	// chrootServePath is the path to server if we are performing a chroot
 	// this probably does not belong here.
-	CHROOT_SERVE_PATH = "/"
-	// AUTH_TOKEN_HEADER is the custom HTTP Header used
+	chrootServePath = "/"
+	// authTokenHeader is the custom HTTP Header used
 	// to authenticate to image inspector.
-	// We use a custom auth header instead of Authorization
+	// Clients must use a custom auth header
+	// instead of standard Authorization
 	// because Kubernetes Proxy strips the default Auth Header
 	// from requests
-	AUTH_TOKEN_HEADER = "X-Auth-Token"
+	authTokenHeader = "X-Auth-Token"
```

Comment:
@ilackarms these are good changes, if you split them out to another PR will be merged quickly I suppose.

Let's not mix refactoring and new features.